dvc/cli/__init__.py

```diff
@@ -39,12 +39,17 @@ def parse_args(argv=None):
 
 def _log_unknown_exceptions() -> None:
     from dvc.info import get_dvc_info
-    from dvc.logger import FOOTER
+    from dvc.ui import ui
+    from dvc.utils import colorize
 
     logger.exception("unexpected error")
     if logger.isEnabledFor(logging.DEBUG):
         logger.debug("Version info for developers:\n%s", get_dvc_info())
-    logger.info(FOOTER)
+
+    q = colorize("Having any troubles?", "yellow")
+    link = colorize("https://dvc.org/support", "blue")
+    footer = f"\n{q} Hit us up at {link}, we are always happy to help!"
+    ui.error_write(footer)
```

Comment:
Tbh I'd prefer getting rid of this completely.

Reply:
why?

Reply:
- It's noisy
- not all errors are critical or require support. Errors are part of the workflow for the most part. eg:
```console
$ dvc import-url https://data.dvc.org/get-started/data.xml

ERROR: unexpected error - [Errno 17] File exists: 'data.xml'               

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```
- we don't have a way to distinguish if those errors are internal error or just normal UX/workflow. Even more so with fsspec, since they raise `OSError`.

Also see https://github.com/iterative/dvc/issues/4126.

Reply:
Thanks for the details @skshetry .
Feels like most of the reasons are not really about the message itself but the circumstances around it.

I believe it would still be useful if we could show it when it really should

Reply:
Maybe we could add a config to opt-out to begin with

Reply:
yeah, but atm the reality is that it's not helpful. It's not clear to me if that link is helpful to the users.

Reply:
> yeah, but atm the reality is that it's not helpful. It's not clear to me if that link is helpful to the users.

IIn the current state, I am ok with dropping it